application/controllers/timers.py

```diff
@@ -0,0 +1,90 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.timer import Timer
+from application import db
+
+timer_fields = {
+    'id': fields.Integer,
+    'work_interval': fields.String,
+    'rest_interval': fields.String
+}
+
+timer_list_fields = {
+    'count': fields.Integer,
+    'timers': fields.List(fields.Nested(timer_fields)),
+}
+
+timer_post_parser = reqparse.RequestParser()
+timer_post_parser.add_argument(
+    'work_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='work_interval parameter is required'
+)
+timer_post_parser.add_argument(
+    'rest_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='rest_interval parameter is required'
+)
+
+class TimersResource(Resource):
```

Comment:
It's interesting that these controllers seem to be like a model controller hybrid from Rails. @sienna-kopf @Chulstro Does that seem right to you both?